model/qti/metadata/ontology/MappedMetadataInjector.php

```diff
@@ -98,7 +98,9 @@ private function isInjectableProperty(array $mappedProperties, string $mappedPat
 
     private function setListValue(Property $property, Resource $resource, SimpleMetadataValue $metadataValue): void
     {
+        $resource->removePropertyValues($property);
```

Comment:
Good catch!

Indeed it makes sure existing values will be removed and only the new ones will be set, but how does it prevent that the resource property does not get duplicated? 🤔